style/tables.css

```diff
@@ -13,6 +13,13 @@
   box-sizing: border-box;
   position: relative;
 }
+
+.ProseMirror td:not([colwidth]):not(.column-resize-dragging),
+.ProseMirror th:not([colwidth]):not(.column-resize-dragging) {
+  /* if there's no explicit width set and the column is not being resized, set a default width */
+  min-width: var(--default-cell-min-width);
+}
```

Comment:
I don't understand the necessity of `:not(.column-resize-dragging)`. It seems that `:not([data-colwidth])` is enough since all td/th with explicit width have this attribute.